### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by alekskz**
Create:
1. New clsss AddActivity
2. New layout add_activiy
3. New launcher
4. Color / color_button_text
Edit:
1. AndroidManifest
2. Colors

### CODE REVIEW

File: app/src/main/java/kz/aleksey/loftmoney/AddActivity.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            public void onTextChanged(CharSequence s, int start, int before, int count) {}
+
+            @Override
+            public void afterTextChanged(Editable s) {
+               if (nameInput.getText().length() > 0 &&
+                       priceInput.getText().length() > 0) {
+                   addBtn.setEnabled(true);
+               } else {
+                   addBtn.setEnabled(false);
+               }
```
Comment (askden8) [inline app/src/main/java/kz/aleksey/loftmoney/AddActivity.java]: Can be write more compact

File: app/src/main/res/layout/activity_add.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:background="@color/colorWhite"
+    tools:context=".AddActivity">
+
+    <EditText
+        android:id="@+id/name"
+        android:hint="@string/add_hint_name"
+        android:textSize="24sp"
```
Comment (askden8) [inline app/src/main/res/layout/activity_add.xml]: Pls use dimens

**Comment by alekskz** [comment]
Done.

**Comment by alekskz** [comment]
Edited, taking into account the last comments.

**Pull request merged**